command/wrist.py

```diff
@@ -0,0 +1,81 @@
+import utils
+import constants
+import wpilib
+import config
+from oi.keymap import Controllers
+from toolkit.command import SubsystemCommand, radians
+from subsystem import Wrist
+from commands2 import SequentialCommandGroup
+import math
+
+class ZeroWrist(SubsystemCommand[Wrist]):
+
+    def __init__(self, subsystem: Wrist):
+        super().__init__(subsystem)
+        self.subsystem = subsystem
+
+    def initialize(self):
+        self.subsystem.zero_wrist()
+
+    def execute(self):
+        pass
+
+    def isFinished(self):
+        return self.subsystem.is_at_angle(0)
```

Comment:
Just return true, since we do not want to set the rotation to zero straight away

Reply:
Why don't we want to set the rotation to zero straight away? I ask because this is how it was written in the charged up offseason repo